guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>The output from <code>lr_find()</code> can be used while fitting the model, however, a different learning rate, determined by interpreting the graph, can be used for model training. You can also skip using <code>lr_find()</code> and instead directly jump to <code>fit()</code> by specifying the number of epochs. In this case, the API will internally call the <code>lr_find()</code> and use the resulting suggested value. </p><p>For example, in the line below, the timeseries model is trained for 50 epochs, while automatically estimating the learning rate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>